### PR TITLE
testkube: cleanup & document support for OpenShift

### PR DESCRIPTION
**Pull request by dejanzele**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: charts/testkube-api/templates/minio.yaml [hunk 1]
```diff
@@ -57,7 +57,7 @@ spec:
             # Name of the PVC created earlier
             claimName: testkube-minio-pv-claim-{{ .Release.Namespace }}
       containers:
-        - name: testkube-minio-{{ .Release.Namespace }}
+        - name: testkube-minio
```
Comment (vsukhin) [inline charts/testkube-api/templates/minio.yaml:60]: will it work for existing minio instance or we will loose previous  data

File: charts/testkube-api/values.yaml [hunk 2]
```diff
@@ -232,8 +224,8 @@ minio:
   image:
     registry: docker.io
     repository: minio/minio
-    tag: latest
-    pullPolicy: Always
+    tag: 2023.2.27
```
Comment (vsukhin) [inline charts/testkube-api/values.yaml:227]: why this particular version?

File: charts/testkube-api/templates/minio.yaml (hunk 1 above)
Comment (dejanzele) [reply charts/testkube-api/templates/minio.yaml:60]: this should only restart the minio pod, data should be on the PVC

File: charts/testkube-operator/templates/role.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@
 apiVersion: {{ include "global.capabilities.rbac.apiVersion" . }}
 kind: ClusterRole
 metadata:
-  name: {{ .Release.Name }}-operator-manager-role
+  name: testkube-operator-manager-role
```
Comment (vsukhin) [inline charts/testkube-operator/templates/role.yaml:7]: should it be hardcoded, user submitted pr and told about non matching?

File: charts/testkube-operator/templates/rolebinding.yaml [hunk 4]
```diff
@@ -4,7 +4,7 @@
 apiVersion: {{ include "global.capabilities.rbac.apiVersion" . }}
 kind: ClusterRoleBinding
 metadata:
-  name: {{ .Release.Name }}-operator-manager-rolebinding
+  name: testkube-operator-manager-rolebinding
```
Comment (vsukhin) [inline charts/testkube-operator/templates/rolebinding.yaml:7]: same question about hard coding

**Review by vsukhin — APPROVED** [review]
looks good, but have a few questions

File: charts/testkube-operator/templates/deployment.yaml [hunk 5]
```diff
@@ -70,26 +72,26 @@ spec:
...
+            port: 8081
+          initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+          periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
         readinessProbe:
           httpGet:
             path: /readyz
-            port: {{ .Values.readinessProbePort }}
-          initialDelaySeconds: {{ .Values.readinessProbeinitialDelaySeconds }}
-          periodSeconds: {{ .Values.readinessProbeperiodSeconds }}
+            port: 8081
```
Comment (vsukhin) [inline charts/testkube-operator/templates/deployment.yaml:81]: why is it hardcoded? users asked about ability to change this

**Review by vsukhin — APPROVED** [review]
looks good

File: charts/testkube-operator/templates/deployment.yaml (hunk 5 above)
Comment (dejanzele) [reply charts/testkube-operator/templates/deployment.yaml:81]: It wasn't configured before to be changeable, i.e. we could change this value but we didnt change the `--health-probe-bind-address=:8081` flag.

If it needs to be configurable we need to change the flag also to support it.

Comment (dejanzele) [reply charts/testkube-operator/templates/deployment.yaml:81]: Ok, I've made it configurable

**Pull request merged**